x/concentrated-liquidity/fees_test.go

```diff
@@ -144,67 +144,111 @@ func (s *KeeperTestSuite) TestInitializeFeeAccumulatorPosition() {
 
 func (s *KeeperTestSuite) TestGetFeeGrowthOutside() {
 	type feeGrowthOutsideTest struct {
-		poolSetup           bool
-		tickSetup           bool
-		feeAccumulatorSetup bool
-		expectedError       bool
+		poolSetup bool
+
+		lowerTick                 int64
+		upperTick                 int64
+		currentTick               int64
+		lowerTickFeeGrowthOutside sdk.DecCoins
+		upperTickFeeGrowthOutside sdk.DecCoins
+		globalFeeGrowth           sdk.DecCoin
+
+		expectedFeeGrowthOutside sdk.DecCoins
+		invalidTick              bool
+		expectedError            bool
 	}
 
+	defaultAccumCoins := sdk.NewDecCoins(sdk.NewDecCoin(ETH, sdk.NewInt(10)))
+	defaultPoolId := uint64(1)
+	defaultInitialLiquidity := sdk.OneDec()
+	// defaultLiquidityForLowerTick := sdk.MustNewDecFromStr("10.0")
+	// defaultLiquidityForUpperTick := sdk.MustNewDecFromStr("20.0")
+
+	defaultUpperTickIndex := int64(5)
+	defaultLowerTickIndex := int64(3)
+
 	tests := map[string]feeGrowthOutsideTest{
-		// TODO: uncomment this once tickInfo feeGrowthOutside logic has been implemented
-		"happy path": {
-			poolSetup:           true,
-			tickSetup:           true,
-			feeAccumulatorSetup: true,
-			expectedError:       false,
-		},
-		// "tick has not been initialized": {
-		// 	poolSetup:           true,
-		// 	tickSetup:           false,
-		// 	feeAccumulatorSetup: true,
-		// 	expectedError:       false,
-		// },
+		"single swap left -> right: 2 ticks, one share - current price > upper tick": {
```

Comment:
Could you please add an explanation comment similar to what you have for left-to-right tests cases?

Reply:
done!